dashboard/app/controllers/projects_controller.rb

```diff
@@ -15,6 +15,9 @@ class ProjectsController < ApplicationController
     playlab: {
       name: 'New Play Lab Project'
     },
+    starwars: {
+      name: 'New Star Wars Project'
```

Comment:
I think so.  It matches the other "Project" level names: `New Artist Project`, `New Play Lab Project`, `New Game Lab Project` etc.